### ISSUE TITLE
Provide aarch64 AppImage ?

### ISSUE THREAD

**Issue by abcfy2** [probonopd/linuxdeployqt#622]
Would you please also provide aarch64 AppImage download in release page: https://github.com/probonopd/linuxdeployqt/releases/tag/continuous 

Thanks.

**Comment by probonopd** [comment]
Hello, I don't actively work on linuxdeployqt anymore.
My newer tool, https://github.com/probonopd/go-appimage, has an aarch64 build. Maybe it is useful for you?

**Comment by abcfy2** [comment]
Hi @probonopd . Thank you very much. But currently go-appimage does not support exclude libs features like linuxdeployqt yet: https://github.com/probonopd/go-appimage/issues/291

Thanks.

**Comment by probonopd** [comment]
I think you can just `rm` unwanted libs after the `deploy` step.

**Comment by probonopd** [comment]
I don't think we have this yet, you could try to copy them manually, but possible then their dependencies don't get deployed properly.

**Comment by abcfy2** [comment]
And how to do `-extra-plugins` in go-appimage ?

**Comment by probonopd** [comment]
I don't think we have this yet, you could try to copy them manually, but possible then their dependencies don't get deployed properly.

**Comment by abcfy2** [comment]
OK, got it. I will try to gradually migrate to go-appimage. But `-exclude-libs` and `-extra-plugins` are very nice features in `linuxdeployqt`. It would be great if `go-appimage` can support these features in the future.

Anyway, thank you for your hard work.
